cleverpush/src/main/java/com/cleverpush/CleverPushPreferences.java

```diff
@@ -19,14 +19,15 @@
     public static final String SUBSCRIPTION_CREATED_AT = "subscriptionCreatedAt";
     public static final String SHOWN_APP_BANNERS = "shownAppBanners";
     public static final String NOTIFICATIONS = "notifications"; // deprecated, now using NOTIFICATIONS_JSON
-    public static final String NOTIFICATIONS_JSON = "notificationsJson";
+	public static final String NOTIFICATIONS_JSON = "notificationsJson";
     public static final String LAST_NOTIFICATION_ID = "lastNotificationId";
     public static final String APP_OPENS = "appOpens";
     public static final String APP_REVIEW_SHOWN = "appReviewShownAt";
     public static final String PENDING_TOPICS_DIALOG = "pendingTopicsDialog";
-    public static final String APP_BANNER_SESSIONS = "appBannerSessions";
-    public static final String APP_BANNERS_DISABLED = "appBannersDisabled";
+	public static final String APP_BANNER_SESSIONS = "appBannerSessions";
```

Comment:
Revert this change.

In AndroidStudio settings please enable Spaces instead of Tabs